attractions/checkbox/checkbox.svelte

```diff
@@ -1,18 +1,58 @@
 <script>
+  /**
+   * @event {{ value: string; checked: boolean; nativeEvent: Event }} change
+   */
   import { createEventDispatcher } from 'svelte';
   import classes from '../utils/classes.js';
 
   let _class = null;
   export { _class as class };
+  /**
+   * A class string to assign to the `<input>` element.
+   * @type {string}
+   */
   export let inputClass = null;
+  /**
+   * A class string to add to the selector box element.
+   * @type {string}
+   */
   export let selectorClass = null;
+  /**
+   * A CSS style string to assign to the selector box element.
+   * Can be used to make the checkbox represent a color (in conjunction with the `getColorPickerStyles` utility).
```

Comment:
Should we link to the utility here?

Reply:
If you mean with the `@see` or `@link` tags, they will not make it through sveld. It only passes the text description and tags it understands through. If you mean a normal markdown link, to what should it link? The docs or the implementation?

Reply:
Yeah, I mean the normal Markdown link. Link to the docs, like we do with any other utility.

Reply:
By the way, there's a typo in the docs, the utility is called `getColorPickerStyles`, not `getColorPickerStyle`